etc/scripts/utils_dejacode.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
```

Comment:
These scripts are present in all the repo in which skeleton is merged. And since we always merge the skeleton to keep it updated these files will be there and we have to remove it. Also, since this will be the last change before deltacode gets archived IMHO it's better if we can keep everything consistent as of now and then archive it. Let me know if I'm missing something. :)

Reply:
@arnav-mandal1234 makes sense, this wouldn't be that important as we're merging into scancode-toolkit soon anyway.